### PR TITLE
Fixes to #9 #10 and small style fixes

### PR DESCRIPTION
**Pull request by ironfroggy**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -3,14 +3,54 @@
...
+    * Copy/pasteable code snippets with syntax highlighting
 
 
 Documentation
 -----------------------------------
 
-Documentation on using django-styleguide is available on 
-`Read The Docs <http://readthedocs.org/docs/django-styleguide/>`_.
+Styleguide provides both a mechanism to create styleguides for areas of your website or types of
+elements in your design and also provides a few started styleguides. To use the starter styleguides,
```
Comment (dpoirier) [inline README.rst]: s/started/starter/ I think.

File: README.rst [hunk 2]
```diff
@@ -3,14 +3,54 @@
...
+first see which ones are present.
+
+    python manage.py listpatterns
+
+The `listpatterns` command will give you a list of the styleguides that come form
```
Comment (dpoirier) [inline README.rst]: s/form/from/

File: README.rst [hunk 3]
```diff
@@ -3,14 +3,54 @@
...
+`django-styleguide` itself. If you want to customize any of these, you can use the `copypattern`
```
Comment (dpoirier) [inline README.rst]: Does `listpatterns` not show any patterns you've copied and customized?

File: styleguide/management/commands/copypattern.py [hunk 4]
```diff
@@ -17,6 +17,8 @@ def handle_label(self, label, **options):
         except (KeyError, IndexError):
             raise CommandError("Project template DIRS not found")
         dest_dir = os.path.join(project_templates, "styleguide")
+        if not os.path.exists(dest_dir):
+            os.mkdir(dest_dir)
```
Comment (dpoirier) [inline styleguide/management/commands/copypattern.py]: `os.makedirs(dest_dir)` would work even if the parent directories don't exist.

File: styleguide/static/styleguide/styleguide.css [hunk 5]
```diff
@@ -1,3 +1,11 @@
+/* TOOD GET RID OF */
```
Comment (dpoirier) [inline styleguide/static/styleguide/styleguide.css]: s/TOOD/TODO/ ?

File: styleguide/templates/styleguide/styleguide-form.html [hunk 6]
```diff
@@ -43,6 +43,11 @@
         <span class=error>This field has an invalid state</span>
     {% endexample %}
 
+
+        {% example "Disabled" %}
+            <input type=text disabled value="This cann't be changed"></input>
```
Comment (dpoirier) [inline styleguide/templates/styleguide/styleguide-form.html]: s/cann't/can't/

File: styleguide/urls.py [hunk 7]
```diff
@@ -2,9 +2,9 @@
 from . import views
 
 
-urlpatterns = patterns('',
+urlpatterns = [
```
Comment (dpoirier) [inline styleguide/urls.py]: Does this make the `patterns` import unused?

File: README.rst [hunk 8]
```diff
@@ -3,14 +3,54 @@
 django-styleguide
 ========================
 
-An app to manage a live style guide in your Django project.
+Welcome to the documentation for django-styleguide!
+
+**Vision**
```
Comment (dpoirier) [inline README.rst]: Can we use RST headers here?

File: README.rst (hunk 3 above)
Comment (ironfroggy) [inline README.rst]: Currently no, its intended to show the ones that come built in so you can copy and customize them.

File: styleguide/templates/styleguide/styleguide-form.html (hunk 6 above)
Comment (ironfroggy) [inline styleguide/templates/styleguide/styleguide-form.html]: So I can't fix that, it says it can't be changed.

**Comment by ironfroggy** [comment]
I have addressed all comments in the latest commits

**Comment by dpoirier** [comment]
I don't have a way to test the style changes, but the doc changes look good.

**Comment by nmashton** [comment]
I agree, looks good. :guitar:

**Pull request merged**